pulsar-common/src/test/java/org/apache/pulsar/common/util/collections/DefaultRangeSetTest.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.common.util.collections;
+
+import org.testng.annotations.Test;
+
+import static org.testng.AssertJUnit.assertNull;
+import static org.testng.AssertJUnit.fail;
+
+public class DefaultRangeSetTest {
+    static final LongPairRangeSet.LongPairConsumer<LongPairRangeSet.LongPair> consumer =
+            LongPairRangeSet.LongPair::new;
+
+    @Test
+    public void testBehavior() {
+        LongPairRangeSet.DefaultRangeSet<LongPairRangeSet.LongPair> set =
+                new LongPairRangeSet.DefaultRangeSet<>(consumer);
+        ConcurrentOpenLongPairRangeSet<LongPairRangeSet.LongPair> rangeSet =
+                new ConcurrentOpenLongPairRangeSet<>(consumer);
+
+        assertNull(set.firstRange());
+        assertNull(set.lastRange());
+        assertNull(set.span());
+
+        assertNull(rangeSet.firstRange());
+        assertNull(rangeSet.lastRange());
+        assertNull(rangeSet.span());
+
+        try {
+            set.contains(null);
```

Comment:
can you please add a positive test ?
something that calls "contains" and return the expected value?

because only testing for a NPE may simply say that this method always throws NPE :-)

Reply:
OK, other unit tests have covered normal scenarios, I will add them here too